lib/routes/dytt/index.ts

```diff
@@ -0,0 +1,76 @@
+import type { Route, Context } from '@/types';
+import got from '@/utils/got';
+import { load } from 'cheerio';
+import iconv from 'iconv-lite';
+import { parseDate } from '@/utils/parse-date';
+import cache from '@/utils/cache';
+
+export const route: Route = {
+    path: '/',
+    categories: ['multimedia'],
+    example: '/dytt',
+    parameters: {},
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: true,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['ygdy8.net/index.html'],
+        },
+    ],
+    name: '电影天堂',
```

Comment:
Route name should not be the same as namespace name https://github.com/DIYgod/RSSHub/blob/20704224caf724933edfdca5bcb232134f3b3a61/lib/routes/dytt/namespace.ts#L4